meeting-notes/2022-05-26.md

```diff
@@ -0,0 +1,115 @@
+# Thursday 5/26/2022
+
+@phillco, @pokey
+
+Note: this session is pretty heavily focused on JetBrains, but substitute with "
+the non VS Code editor" wherever appropriate.
+
+## What's working today?
+
+- A new JetBrains plugin in Kotlin that uses event listeners, and serializes
+  editor state to a file (namespaced by PID) for each JetBrains instance.
+- This should have long term benefits, beyond just Cursorless, because Talon can
+  now simply read the PID state file for the current editor to find information
+  about it. (Today, everything has to go through synchronous HTTP requests
+  JetBrains, and sometimes those block if UI is showing.).
+    - It could also watch it to be proactively notified about state changes.
```

Comment:
Worth thinking about race conditions here, but maybe ok for intermediate version before we get proper talon RPC